package.json

```
@@ -49,7 +49,7 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "dependencies": {
-    "@babel/parser": "7.16.4",
+    "@babel/parser": "7.21.3",
     "@babel/traverse": "^7.12.5",
```

```suggestion
    "@babel/traverse": "7.22.5",
```

Maybe update that one too for consistency.

---

Same as `@babel/parser`, I did this manually to update lock file.